nemo_skills/training/nemo_rl/start_sft.py

```
@@ -98,6 +101,13 @@ def __init__(
         else:
             self.num_proc = num_proc
 
+        self.input_template = None
+        if input_template_path:
+            input_template_config = load_prompt_config(input_template_path)
+            if "user" not in input_template_config:
+                raise KeyError(f"'user' key is missing in the YAML file: {input_template_path}")
```

Ruff (0.13.3)

Avoid specifying long messages outside the exception class

(TRY003)